pubsub/cloud-client/publisher_test.py

```diff
@@ -45,6 +46,20 @@ def topic(client):
     yield response.name
 
 
+@pytest.fixture
+def topic_two(client):
```

Comment:
When is `topic_one` used, and when is `topic_two` used? Can we change their names to something that gives us a better idea of what they're for?

Reply:
Done.